package.json

```diff
@@ -15,33 +15,33 @@
   "author": "Dynamo Team",
   "license": "MIT",
   "dependencies": {
+    "@types/jest": "^20.0.0",
+    "babel-jest": "^20.0.3",
     "express": "^4.14.0",
+    "jest": "^20.0.4",
```

Comment:
Can we put these few dependencies to `devDependencies`, which means we should install these dependencies by commands like `npm install jest --save-dev`